bokehjs/src/lib/models/annotations/legend.ts

```diff
@@ -386,17 +387,34 @@ export class LegendView extends AnnotationView {
       }
     })()
 
+    const has_background = (_i: number, row: number, col: number) => {
```

Comment:
suggest `has_item_background` as more descriptive. That would core closely match both the implementation that checks `item_background_policy` and the call side where item backgrounds are drawn.